package.json

```diff
@@ -1,11 +1,21 @@
 {
   "name": "root",
-  "version": "1.36.0-next.0",
+  "version": "1.36.0-next.2",
```

Comment:
really?

Reply:
Not quite sure why GitHub is showing this, it's the current version O_o